index.html

```diff
@@ -1346,26 +1338,25 @@
       </h2>
       <pre class="idl">
         dictionary PaymentDetailsModifier {
-          required sequence&lt;DOMString&gt; supportedMethods;
+          required DOMString supportedMethods;
           PaymentItem total;
           sequence&lt;PaymentItem&gt; additionalDisplayItems;
           object data;
         };
       </pre>
       <p>
         The <a>PaymentDetailsModifier</a> dictionary provides details that
-        modify the <a>PaymentDetailsBase</a> based on <a>payment method
-        identifier</a>. It contains the following fields:
+        modify the <a>PaymentDetailsBase</a> based on a <a>payment method
+        identifier</a>. It contains the following members:
       </p>
       <dl>
         <dt>
           <dfn>supportedMethods</dfn>
```

Comment:
> Is this intentional? Changing the name would be more of a clean slate, but I suppose risks ending up having to support both.

Yeah, it's intentional. It's probably as backwards compatible as we are going to get.